cli/command/image/push.go

```diff
@@ -11,26 +11,34 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type pushOptions struct {
+	remote    string
+	untrusted bool
```

Comment:
Can we somehow flip this boolean? I'm a bit ... erm ... idk about having `untrusted`, and it looks like _everywhere_ we use it as `if !untrusted`

Reply:
I don't think so. The flag is `disable-content-trust`, so it has to store true to disable it.

Reply:
Yeah, it's a weird flag;

```
docker run --help

--disable-content-trust          Skip image verification (default true)
```


```
DOCKER_CONTENT_TRUST=1 docker run --help

--disable-content-trust          Skip image verification
```